crawler.js

```diff
@@ -912,12 +912,14 @@ export class Crawler {
     const realSize = await this.crawlState.queueSize();
     const pendingList = await this.crawlState.getPendingList();
     const done = await this.crawlState.numDone();
-    const total = realSize + pendingList.length + done;
+    const failed = await this.crawlState.numFailed();
+    const total = realSize + pendingList.length + done + failed;
```

Comment:
Only thought is: do we want `done` to store only succeeded, and not failed, or include the failed ones?
Could go either way, i guess for backwards compatibility maybe slight preference for done = succeeded + failed.
In that case, the `movefailed` should also include the incr command?

Reply:
Change made, good point!

Reply:
yup, sorry!